src/modules/lv2/filter_lv2.yml

```diff
@@ -0,0 +1,25 @@
+schema_version: 7.0
+type: filter
+identifier: lv2
+title: LV2
+version: 1
+license: GPLv2
+language: en
+url: http://www.lv2.org/
+creator: Dan Dennedy
+tags:
+  - Audio
+description: Process audio using LV2 plugins.
+notes: >
+  Automatically adapts to the number of channels and sampling rate of the consumer.
+bugs:
+  - Some effects have a temporal side-effect that may not work well.
+
+parameters:
+  - identifier: resource
+    argument: yes
+    title: JACK Rack XML file
```

Comment:
Does it really handle a JACK Rack XML file?

Reply:
No